### ISSUE TITLE
TODO: Softmax

### ISSUE THREAD

**Issue by canyalniz** [canyalniz/CNN-Inference-Didactic#3]
[Core Keras Activation Softmax](https://keras.io/activations/#softmax) must be implemented and documented.
